packages/travix-ui-kit/components/slidingPanel/slidingPanelHeader/slidingPanelHeader.js

```diff
@@ -3,20 +3,54 @@ import React from 'react';
 import classnames from 'classnames';
 import { getDataAttributes } from '../../_helpers';
 
-const renderDefaultLeftBlock = (backButtonLabel, onBackButtonClick) => (
-  <button
-    className="ui-sliding-panel-header__left-block-back"
-    onClick={onBackButtonClick}
-  >
-    <span className="ui-sliding-panel-header__left-block-back-icon" />
-
-    { backButtonLabel && (
-      <span className="ui-sliding-panel-header__left-block-back-text">
-        {backButtonLabel}
-      </span>
-    ) }
-  </button>
-);
+const renderLeftBlock = (
+  backButtonLabel,
+  leftBlock,
+  onBackButtonClick,
+  useDefaultLeftBlock,
```

Comment:
For me it looks a bit starange to pass function, which returns `null`, to prevent default element rendering. `useDefaultLeftBlock` is more obvious option, from my point of view.

Reply:
There are some extra settings for clearance of UI-user.
By default, the `default block` is not rendered. If we remove this behaviour:
1. it will be breaking change
2. There is no benefit for users since you force them to pass a function to render null all the time
3. The function, which purpose is to return null is useless and its existence only shows the bad design.

So, I do not agree that rendering empty function is a good option.

Reply:
@rbardini removing `useDefaultLeftBlock` will be a breaking change. You can create an issue for next major release and get feedback there.

Reply:
What about deprecating `useDefaultLeftBlock` in favor of `renderLeftBlock` then? We could even accept boolean values in render props (including `renderRightBlock`), so if you don't want to render the default element you can declare `renderLeftBlock={false}`. This way we keep it backwards compatible and ease the transition to render props.

Reply:
@rbardini, I would recommend to follow @asci suggestion. You can create issue for that, describe possible approaches, which you already have, and community will be able to decide, what to do.

Reply:
I'll open an issue once the PR is merged.